py/src/nupic/encoders/grid_cell_encoder.py

```diff
@@ -0,0 +1,136 @@
+# ------------------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+#
+# Copyright (C) 2018-2019, David McDougall
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero Public License version 3 as published by the Free
+# Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License along with
+# this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ------------------------------------------------------------------------------
+
+
+import numpy as np
+import math
+from nupic.encoders import hexy
+from nupic.bindings.sdr import SDR
+from nupic.bindings.math import Random
+
+class GridCellEncoder:
+    """ TODO: DOCUMENTATION """
```

Comment:
please put some doc here, even a link to your repo with demonstration and a few lines explaining what this does..